src/checkpoint/checkpoint_stats.c

```diff
@@ -23,13 +23,32 @@ __wt_checkpoint_reset_handle_stats(WT_SESSION_IMPL *session, WT_CKPT_CONNECTION
 }
 
 /*
- * __wt_checkpoint_set_handle_stats --
- *     Set handle-related stats.
+ * __wt_checkpoint_reset_time_stats --
+ *     Reset time-related stats.
  */
 void
-__wt_checkpoint_set_handle_stats(
-  WT_SESSION_IMPL *session, WT_CKPT_CONNECTION *ckpt, uint64_t gathering_handles_time_us)
+__wt_checkpoint_reset_time_stats(WT_SESSION_IMPL *session, WT_CKPT_CONNECTION *ckpt)
 {
+    WT_UNUSED(session);
+
+    ckpt->prepare.min = UINT64_MAX;
+    ckpt->ckpt_api.min = UINT64_MAX;
+    ckpt->scrub.min = UINT64_MAX;
+}
+
+/*
+ * __wt_checkpoint_stats_handle_update --
+ *     Update handle-related stats.
+ */
+void
+__wt_checkpoint_stats_handle_update(WT_SESSION_IMPL *session, uint64_t gathering_handles_time_us)
+{
+    WT_CONNECTION_IMPL *conn;
+    WT_CKPT_CONNECTION *ckpt;
```

Comment:
Alphabetize please. Update: See comment below about getting rid of `conn`.

Reply:
I followed your advice about removing the ckpt arg; it's much cleaner! See d4dce6e7d0dd719e8eb5c24fe5b9f138555f7a82